idea-plugin/src/main/java/com/palantir/javaformat/intellij/FormatterProvider.java

```diff
@@ -146,13 +150,13 @@ private static Integer parseSdkJavaVersion(Sdk sdk) {
     }
 
     @VisibleForTesting
-    static Integer parseSdkJavaVersion(String version) {
+    static OptionalInt parseSdkJavaVersion(String version) {
```

Comment:
It looks like the Optional-ness of these OptionalInt return values is never used -- it's either a value or null.

Reply:
Fixed